packages/blocks/src/root-block/edgeless/edgeless-root-block.ts

```diff
@@ -126,6 +126,13 @@ export class EdgelessRootBlockComponent extends BlockElement<
     }
   `;
 
+  /**
+   * Disable components
+   *
+   * Toolbar is not allowed to display in `syncd doc block`.
+   */
+  disableComponents = false;
+
```

Comment:
The edgeless toolbar component will be converted to a widget in this cycle. After that, I think this property will no longer be necessary.

Does this PR need to be merged as soon as possible?

Reply:
> The edgeless toolbar component will be converted to a widget in this cycle. After that, I think this property will no longer be necessary.

Yes, that's a workaround.

Make it a widget that will be great.
I feel like we can merge first.